MatrixSDK/Crypto/CryptoMachine/MXCryptoMachine.swift

```diff
@@ -510,6 +546,85 @@ extension MXCryptoMachine: MXCryptoSASVerifying {
     }
 }
 
+extension MXCryptoMachine: MXCryptoBackup {
+    var isBackupEnabled: Bool {
+        return machine.backupEnabled()
+    }
+    
+    var backupKeys: BackupKeys? {
+        do {
+            return try machine.getBackupKeys()
+        } catch {
+            log.error("Failed fetching backup keys", context: error)
```

Comment:
Does this need to be handled better, throw or maybe just crash?

Reply:
I don't know what legitimate reasons for this to fail are, given that it still returns an optional if we don't have any backup. It is called by too many places to surface the error upward so logs / sentry will have to suffice